src/Controllers/WishlistController.php

```diff
@@ -46,7 +46,9 @@ public function store(AuthenticatedRequest $request): mixed
         $rapidezWishlist->save();
 
         if (!Wishlist::where('customer_id', $request->customer_id)->first()) {
-            $magentoWishlist = new Wishlist($validated);
+            $magentoWishlist = new Wishlist([
```

Comment:
Maybe it's nicer to use https://laravel.com/docs/9.x/helpers#method-array-only and the $validated array just to ensure the `$validated` array will always be used, even when new data may be added in the future